content/departments/marketing/blog.md

```diff
@@ -37,7 +37,13 @@ Feel free to add many code snippets and use code repositories to complement your
 
 ## Adding a blog post
 
-Once your draft is approved, you're ready to add a blog post by creating a Markdown file in one of the `sourcegraph/about` repository `content/blogposts` child directories. You can use [Docs to Markdown](https://workspace.google.com/u/0/marketplace/app/docs_to_markdown/700168918607) to make that conversion and then use the following template for your front matter:
+Once your draft is approved, you're ready to add a blog post by creating a Markdown file in one of the `sourcegraph/about` repository `content/blogposts` child directories.
+
+## Convert to markdown
+
+To quickly convert your document into Markdown, you can use [Docs to Markdown](https://workspace.google.com/u/0/marketplace/app/docs_to_markdown/700168918607). This is not required, but can save some time if you do not prefer to rewrite the content.
+
+If you do this, please add the following frontmatter to the beginning of the file:
```

Comment:
I would update this and move the frontmatter to the blog starter pack and use that as the source of truth for all formatting